openbrews/recipe/add/hop.html

```diff
@@ -0,0 +1,40 @@
+<div class="list list-inset">
```

Comment:
Hey, I just checked it, mine looks fine:

![image](https://cloud.githubusercontent.com/assets/3668034/19152748/1ad05096-8bf1-11e6-999d-ee12b11f72de.png)

I have used the exact same DOM as the fermentable structure.

![image](https://cloud.githubusercontent.com/assets/3668034/19152790/4c2ed356-8bf1-11e6-8de2-d0c42e1e3a5d.png)

Both the templates for fermentable and hop start with `<div class="list list-inset">`. Are you seeing the same thing in the fermentable section as well? Or only in the hop section?